### DISCUSSION TITLE
Access-Point | Detecting new connected Devices without http or so?

### DISCUSSION THREAD

**Discussion by WaGi-Coding** [micropython/micropython#9228] [Using MicroPython]
Is it somehow possible to fire an action in micropython when a device connected to the Access Point? Without calling a site or requiring to do something more than connecting on the device?

**Comment by Lobo-T** [comment]
https://docs.micropython.org/en/latest/library/network.html#network.AbstractNIC.status

**Reply by WaGi-Coding** [reply]
> https://docs.micropython.org/en/latest/library/network.html#network.AbstractNIC.status

hmm, from what i see i could not see only ones which also authed properly right? Like it will also list ones attempting to connect/auth

**Reply by Lobo-T** [reply]
I don't know.  You will have to test.
